Makefile

```diff
@@ -36,7 +36,8 @@ TARGETS := $(MAKE_TARGETS) $(CMD_TARGETS)
 
 DOCKER_TARGETS := $(patsubst %,docker-%, $(TARGETS))
 .PHONY: $(TARGETS) $(DOCKER_TARGETS)
-DOCKERFILE_DEVEL := $(CURDIR)/deployments/container/Dockerfile.devel
+DOCKERFILE_DEVEL := "images/devel/Dockerfile"
+K8S_TEST_INFRA := "https://github.com/NVIDIA/k8s-test-infra.git"
```

Comment:
Should these be `?=` so we can overwrite them when invoking `make` (if desired)?